app/components/candidate_interface/restructured_work_history/gap_component.html.erb

```diff
@@ -1,6 +1,10 @@
 <%= govuk_inset_text(classes: 'app-inset-text--important govuk-!-margin-top-0 govuk-!-margin-bottom-0') do %>
   You have a break in your work history (<%= format_months_to_years_and_months(@break_period.length) %>)
   <br>
-  <%= govuk_link_to 'Add another job', candidate_interface_new_restructured_work_history_path %> or
-  <%= govuk_link_to 'add a reason for this break', candidate_interface_new_restructured_work_history_break_path(start_date: @break_period.start_date, end_date: @break_period.end_date) %>
+  <%= govuk_link_to candidate_interface_new_restructured_work_history_path do %>
+    Add another job<span class="govuk-visually-hidden"><%= between_formatted_dates %></span><% end %>
```

Comment:
Need to ensure there’s a space between words in the sentence when read out loud.

```suggestion
    Add another job<span class="govuk-visually-hidden"> <%= between_formatted_dates %></span><% end %>
```

Reply:
Sorry about that, it didn't make any difference on the front end so missed it! Should be g2g now 👍

Reply:
I think it may have done actually… need to view raw page source and not inspect the HTML - no space between the words.